src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -41,24 +42,65 @@ public class RevitDynamoModel : DynamoModel
         #region Events
 
         public event EventHandler RevitDocumentChanged;
```

Comment:
Sorta out of the scope, but it looks like we could fix this.  Can we tell the subscriber anything more about the situation? It imagine a client might want to know the old and new doc.